main.py

```diff
@@ -1,10 +1,53 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players_data = json.load(file)
+
+    for nick_name, player in players_data.items():
+        new_guild = None
+        race_name = player["race"]["name"]
+        race_description = player["race"]["description"]
+        if not Race.objects.filter(name=race_name).exists():
+            new_race = Race(name=race_name, description=race_description)
+            new_race.save()
+        else:
+            new_race = Race.objects.get(name=race_name)
```

Comment:
You can use get_or_create instead 
https://docs.djangoproject.com/en/4.2/ref/models/querysets/#get-or-create
Fix in all places